modules/examiner/php/editexaminer.class.inc

```diff
@@ -112,19 +112,13 @@ class EditExaminer extends \NDB_Form
     {
         $DB = \Database::singleton();
 
+        // get the examinerID
+        $examinerID = $this->identifier;
         foreach ($values['certStatus'] as $testID => $certStatus) {
 
             $date_cert = $values['date_cert'][$testID];
             $comment   = trim($values['comment'][$testID]);
 
-            // get the examinerID - edit_examiner passes the ID through
-            // the identifier, training passes it as a value
```

Comment:
My impression is this comment is referring to two modules: `edit_examiner` and the deleted `training` module.

Since only `edit_examier` now exists, I declared `examinerID` above using the "Edit examiner" method.